photon-lib/src/main/scala/com/linkedin/photon/ml/util/Linalg.scala

```diff
@@ -55,13 +55,38 @@ object Linalg {
     DenseVector(bArr)
   }
 
+  /**
+   * Inverts the matrix $A$, given the Cholesky factorization $l$ of $A$.
+   *
+   * @param l the Cholesky factorization of matrix A
+   * @return the solution
```

Comment:
Capitalize 'the' please for both lines above (for consistency, but mostly for me)

Reply:
Will do.